deploy/helm/secret-operator/templates/daemonset.yaml

```diff
@@ -74,6 +74,27 @@ spec:
               mountPath: /registration
             - name: csi
               mountPath: /csi
+      initContainers:
```

Comment:
I would do this in https://github.com/stackabletech/secret-operator/blob/6ff521b4ccd9db33346767d8183b3ebee4f69624/rust/operator-binary/src/main.rs#L87-L92 rather than clog up the pods with migration initcontainers.

Reply:
My though process was to make it as easy as possible - for implementation as well as when removing it later on.
But I'm also fine with the Rust solution, will change the PR

Reply:
`[src/main.rs:87] &csi_endpoint = "/csi/csi.sock"`
We want to migrate the *registration sock*, which the secret-op doesn't even mount.
Honestly i would simply stick with the init container. This way we also don't CrashLoop the registrar image as the init container *always* runs be before the registrar container

Reply:
Oh, right. I guess we're kind of stuck like this then.